steamed_hams.py

```diff
@@ -172,7 +172,7 @@ def steamed_hams_parser():
     opts = python_opts.format(module=mod)
     modules[mod] = "python3.9 {opts} {module}/{module}.py".format(opts=opts, module=mod)
 
-modules['realtime'] = "source borealisrt_env/bin/activate;" + modules['realtime']
+modules['realtime'] = "source borealisrt_env3.9/bin/activate;" + modules['realtime']
```

Comment:
sort of a magic number, can use a const at the top: PYTHON_VERSION="3.9" or so. Same with isntall_radar_deps.py